cmd/tracee-ebpf/flags/containers.go

```diff
@@ -41,7 +41,15 @@ func contains(s []string, val string) bool {
 
 func PrepareContainers(containerFlags []string) (runtime.Sockets, error) {
 	if len(containerFlags) == 0 {
-		return autoDiscoverSockets(), nil
+		return runtime.Autodiscover(func(err error, runtime runtime.RuntimeId, socket string) {
+			if debug.Enabled() {
+				if err != nil {
+					fmt.Fprintf(os.Stderr, "RuntimeSockets: failed to register default %s socket:\n%v\n", runtime.String(), err)
+				} else {
+					fmt.Fprintf(os.Stdout, "RuntimeSockets: registered default %s runtime socket from %s\n", runtime.String(), socket)
+				}
+			}
```

Comment:
This might be missed if merged before #2110 
@geyslan FYI

Reply:
I don't think this is touched in his PR but true. These should be converted to `log.Debug(...)`.